Classes/Controller/ModuleController.php

```diff
@@ -415,10 +407,11 @@ public function fileReferenceCreateAction($file, $element)
     public function countAction(ServerRequestInterface $request, ResponseInterface $response)
     {
         $requestParameters = $request->getQueryParams();
```

Comment:
Not sure if this is problematic conerning SQL injection. Will talk to Olly...

Reply:
@ohader please have a look at this lines here

Reply:
* `addslashes` was wrong before - it made things more difficult, but did not protect agains SQL injection
* `TYPO3\CMS\Core\Database\Connection::count($item, $tableName)`
** is using `QueryBuilder` internally
** `$tableName` is passed through `Connection::quoteIdentifier`

Conclusion: remove `addslashes` here

Reply:
I removed addslashes and added a comment about this.